modGui/build.xml

```diff
@@ -10,16 +10,25 @@
     <property name="dir.integtest" value="./build/integration-test"/>
     <property name="test.report.dir" value="report"/>
     <property name="dir.javaAPIlib" value="../javaAPIlib"/>
+	<property name="native.src.dir" value="../native" />
+	<property name="native.dir" value="native" />
 
     <!-- Dependencies for building production code (i.e. non-test files) -->
     <path id="classpath.dependency">
         <pathelement location="${dir.mod}/modLogger.jar"/>
         <pathelement location="${dir.mod}/modMcf.jar"/>
         <pathelement location="${dir.mod}/modAionImpl.jar"/>
+        <pathelement location="${dir.mod}/modCrypto.jar"/>
+        <pathelement location="${dir.mod}/modAionBase.jar"/>
         <pathelement location="${dir.lib}/../aion_api/pack/modAionApi.jar" />
```

Comment:
Since aion_api is its own Git repo, it seems strange for its build.xml to the aion/mod folder.  I thought it was better that aion_api's build.xml not assume that it is being called from aion's build.xml.

For now, modified aion's build.xml to copy from aion_api/pack/modAionApi.jar to the mod directory, and modGui's build.xml references that.  Let me know if you see any problems with that.